diff --git a/configure.ac b/configure.ac
index 87b702732..7eaf0d15a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -548,18 +548,19 @@ if test "x$enable_python" != "xno"; then
 		AC_MSG_ERROR([Python support explicitly requested, but no suitable Python interpreter could be found])
 	fi
 
-	PKG_CHECK_MODULES(PYTHON, [pygobject-3.0 >= $PYGOBJECT_REQUIRED],
-			[have_pygobject=yes], [have_pygobject=no])
+	# PKG_CHECK_MODULES(PYTHON, [pygobject-3.0 >= $PYGOBJECT_REQUIRED],
+	# 		[have_pygobject=yes], [have_pygobject=no])
+	#have_pygobject=yes
 
-	if test "x$have_pygobject" = "xno" -a "x$enable_python" = "xyes"; then
-		AC_MSG_ERROR([Python support explicitly requested, but pygobject-3.0 could not be found])
-	fi
+	#if test "x$have_pygobject" = "xno" -a "x$enable_python" = "xyes"; then
+	#	AC_MSG_ERROR([Python support explicitly requested, but pygobject-3.0 could not be found])
+	#fi
 
-	if test "x$have_python" = "xyes" -a "x$have_pygobject" = "xyes"; then
+	#if test "x$have_python" = "xyes" -a "x$have_pygobject" = "xyes"; then
 		pyoverridesdir=`$PYTHON -c "import gi; print(gi._overridesdir)"`
 		AC_SUBST(pyoverridesdir)
 		build_python=yes
-	fi
+	#fi
 fi
 
 AM_CONDITIONAL(ENABLE_PYTHON, test "x$build_python" = "xyes")
