diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2b64423a4..1ecddce14 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -74,10 +74,10 @@ set(TR_NAME ${PROJECT_NAME})
 
 
 # these should be the only five lines you need to change
-set(TR_VERSION_MAJOR "4")
+set(TR_VERSION_MAJOR "3")
 set(TR_VERSION_MINOR "0")
 set(TR_VERSION_PATCH "0")
-set(TR_VERSION_BETA_NUMBER "1") # empty string for not beta
+set(TR_VERSION_BETA_NUMBER "") # empty string for not beta
 set(TR_VERSION_DEV FALSE)
 
 # derived from above: release type
